02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,79 @@
+      places, joining a coding academy, turning my plates of food into casseroles
+      and petting all dogs in sight. In five years I see myself living in New York
+      City and pursuing a career in web development. No other subject has captivated
+      me more and I plan to achieve all my goals that involve this topic. In the
+      past several years I have learned that if you want something in life, you
+      can have it. I've learned this from the situations I've been in, goals that
+      I have conquered and even the women that I have dated. Mistakes have always
+      taught me lessons and from that I have grown into the adult that I want to be.
+      That's me.
+    </p>
+      <div><div>
+  <h2>Hobbies</h2>
+     <ul>
+       <li>DRUMS! And exloring new music and sounds.</li>
+       <li>Making kombucha</li>
+       <li>Socializing and meeting new people</li>
+       <li>Expressing my newfound love for coding.</li>
+       <li>Exercise and healthy eating - <u>treat your body right.</u></li>
+       <li>Spending time with my beautiful girlfriend, Marianna.</li>
+       <li>Getting traditional artwork tattooed - it's quite the addiction.</li>
+       <li>Museums! - My hometown, <strong>Houston</strong>, has some of the best in the country.</li>
+       <br><br>
+ <h2>Things I can live without</h2>
+     <ul>
+       <li>Racism</li>
+       <li>Homophobia - <em>love is love.</em></li>
+       <li>Dishonesty</li>
+       <li>Single strands of hair</li>
+       <li>The end of a warm beer</li>
+       <li>Mushrooms - it's a texture thing.</li>
+       <li>Donald Trump</li>
+       <div><div>
+<table>
+  <thead>
+    <th colspan="3"><u>This is my life.</u></th>
+  </thead>
+  <tbody>
+  <tr>
+    <td><img src="https://scontent-atl3-1.xx.fbcdn.net/v/t1.0-9/13267728_10209985199612553_4567071495800128847_n.jpg?oh=6053422d0a96c615c35c5e43a8b7179a&oe=581A358E"/>
+    </td>
+    <td><img src="https://scontent-atl3-1.xx.fbcdn.net/v/t1.0-0/p206x206/12039610_10208124009963975_1509911422267794328_n.jpg?oh=67877fc30d565b77788e2dba8304727f&oe=585B6A66"/>
+    </td>
+    <td><img src="https://scontent-atl3-1.xx.fbcdn.net/v/t1.0-9/12472474_10209483389867623_6577527649252181555_n.jpg?oh=9e50b58fed62d01a107bab1c39b9b96d&oe=5811E46B"/>
+    </td>
+  </tr>
+  <tr>
+    <td><img src="https://67.media.tumblr.com/6b512fd56e4186207958cef047c91f14/tumblr_o9e6crxnwB1qf34gmo4_500.png"/>
+    </td>
+    <td><img src="https://scontent-atl3-1.xx.fbcdn.net/v/t1.0-9/11215820_10208099441909789_5466287493261420981_n.jpg?oh=e988ffd7bd98c2522368922dba34b630&oe=585BE277"/>
+    </td>
+    <td><img src="https://scontent-atl3-1.xx.fbcdn.net/v/t1.0-9/12141671_10208308344452222_8626648261313971173_n.jpg?oh=dca50f072e0527d93c7b61ba7d791f19&oe=5814D264">
+    </td>
+  </tr>
+</tbody>
+</table>
+<h2>Websites I like</h2>
+<ul>
+  <li><a href="http://tumblr.com">Tumblr</a></li>
+  <li><a href="http://reddit.com">Reddit</a></li>
 </body>
 </html>
```

Comment:
Fantastic work here, Meghan! LOVE it. Your personality and spirit really shine through! Keep it up.